Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 
+gem 'ffi-locale', github: 'k3rni/ffi-locale'
```

Comment:
Add this to the gemspec, as runtime dependency.

Reply:
I tried this first, but it did not work somehow... How can I add a GitHub dependency like this? 🤔

Reply:
Bundle is the ONLY way to add such a GitHub dependency, but those dependencies won't be installed by users while `gem install`ing. So, the way to go is to have stable released dependencies. You can specify a specific range of versions though, in the gemspec file.

Reply:
According to the ffi-locale project page:

| You need to install the GitHub version of this gem, because it was never pushed to RubyGems due to naming conflicts. RubyGems has seanohalpin's very similar gem under this name. Check for that before reporting errors.

It is a bummer that Ruby still has no proper collation of strings by itself.

I probably have to go with https://github.com/twitter/twitter-cldr-rb which is quite heavy in comparison to the simple ffi solution... ?!

Reply:
Oh. Or we could just have a `monkeys.rb` with monkey-patches for the string. We can definitely provide attribution in the README. BTW, why exactly is this dependency required?

Reply:
The gem provides the setlocale and strcoll C function entry points, which is used for proper locale specific sorting (collation) of strings.

For example, in en_US, "e" should be sorted before "E", in Polish "ł" should be sorted between "l" and "m", in German umlauts should be sorted after their corresponding non-umlaut version et cetera.

Reply:
Hmm, can you check if i18n gem solves this issue?

Reply:
Does not look like it would..  😕

It probably is out of scope for a rails-centric library, given that the database usually does the heavy lifting of sorting data.

Reply:
Ah, but I don't think we'll be able to use an unreleased gem. Too bad that rubygems still doesn't have a feature to fetch a gem from github repository itself (like budler). 😞 

The last option - can we drop this special sorting that ffi-locale provides and just do a default Ruby sort?

Reply:
Also, please check once whether [this](https://github.com/seanohalpin/ffi-locale) would do the job. This is the ffi-locale gem that is pushed to rubygems.